pkg/controller/backup_storage_location_controller.go

```diff
@@ -55,7 +55,7 @@ type BackupStorageLocationReconciler struct {
 func (r *BackupStorageLocationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	log := r.Log.WithField("controller", BackupStorageLocation)
 
-	log.Info("Checking for existing backup locations ready to be verified; there needs to be at least 1 backup location available")
+	log.Debug("Verifying availablity of backup storage locations.")
```

Comment:
These new messages ARE better! One suggestion I have is to use "validation" everywhere. For example, in this case this message would be "Validating availability..."/